airbyte-integrations/connectors/source-shopify/source_shopify/source.py

```diff
@@ -117,50 +128,56 @@ def filter_records_newer_than_state(self, stream_state: Mapping[str, Any] = None
             yield from records_slice
 
 
-class Customers(IncrementalShopifyStream):
-    data_field = "customers"
-
-    def path(self, **kwargs) -> str:
-        return f"{self.data_field}.json"
-
-
-class Orders(IncrementalShopifyStream):
-    data_field = "orders"
-
-    def path(self, **kwargs) -> str:
-        return f"{self.data_field}.json"
-
-    def request_params(
-        self, stream_state: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None, **kwargs
-    ) -> MutableMapping[str, Any]:
-        params = super().request_params(stream_state=stream_state, next_page_token=next_page_token, **kwargs)
-        if not next_page_token:
-            params["status"] = "any"
-        return params
-
-
-class ChildSubstream(IncrementalShopifyStream):
+class ShopifySubstream(IncrementalShopifyStream):
     """
-    ChildSubstream - provides slicing functionality for streams using parts of data from parent stream.
+    ShopifySubstream - provides slicing functionality for streams using parts of data from parent stream.
     For example:
        - `Refunds Orders` is the entity of `Orders`,
        - `OrdersRisks` is the entity of `Orders`,
        - `DiscountCodes` is the entity of `PriceRules`, etc.
 
-    ::  @ parent_stream_class - defines the parent stream object to read from
+    ::  @ shopify_stream - defines the parent stream object to read from
     ::  @ slice_key - defines the name of the property in stream slices dict.
     ::  @ nested_record - the name of the field inside of parent stream record. Default is `id`.
     ::  @ nested_record_field_name - the name of the field inside of nested_record.
     ::  @ nested_substream - the name of the nested entity inside of parent stream, helps to reduce the number of
           API Calls, if present, see `OrderRefunds` stream for more.
     """
 
-    parent_stream_class: object = None
+    shopify_stream_class: object = None
     slice_key: str = None
     nested_record: str = "id"
     nested_record_field_name: str = None
     nested_substream = None
 
+    @property
+    def shopify_stream(self) -> object:
```

Comment:
woudl this be a better name? 
```suggestion
    def parent_stream(self) -> object:
```

Reply:
Changed. Done.